src/client/mod.rs

```diff
@@ -161,6 +157,13 @@ pub fn init_fs_client() -> Result<(), Box<dyn std::error::Error>> {
     let _http_manager_address = format!("http://{}", manager_address);
 
     build_hash_ring(config.all_servers_address.clone());
+
+    let mut builder = env_logger::Builder::from_default_env();
+    builder
+        .format_timestamp(None)
+        .filter(None, log::LevelFilter::from_str(&config.log_level).unwrap());
```

Comment:
could you add log_level to cli. when default level is set as info, we could add `--debug` to change log level that would be more flexible.

Reply:
done